docs/src/time_integration.md

```diff
@@ -42,3 +44,179 @@ are the following. Further documentation can be found in the
+Since Trixi.jl [supports exact computation of the Jacobian $J$ by means of automatic differentiation](https://trixi-framework.github.io/Trixi.jl/stable/tutorials/differentiable_programming/), we have access to the Jacobian of a given simulation setup.
+For small (up to approximately $10^4$ DoF) systems, the spectrum $\boldsymbol \sigma = \left \{ \lambda_m \right \}_{m=1, \dots, M}$ can be computed directly using [`LinearAlgebra.eigvals(J)`](https://docs.julialang.org/en/v1/stdlib/LinearAlgebra/#LinearAlgebra.eigvals).
+For larger systems, we recommend the procedure outlined in section 4.1 of [Doehring et al. (2024)](https://doi.org/10.1016/j.jcp.2024.113223). This approach computes a reduced set of (estimated) eigenvalues $\widetilde{\boldsymbol \sigma}$ around the convex hull of the spectrum by means of the [Arnoldi method](https://github.com/JuliaLinearAlgebra/Arpack.jl).
+
+The optimization problem (1) can be solved using the algorithms described in [Ketcheson, Ahmadia (2012)](http://dx.doi.org/10.2140/camcos.2012.7.247) for a moderate number of stages $S$ or [Doehring, Gassner, Torrilhon (2024)](https://doi.org/10.1007/s10915-024-02478-5) for a large number of stages $S$.
+In Trixi.jl, the former approach is implemented by means of convex optimization using the [Convex.jl](https://github.com/jump-dev/Convex.jl) package.
+
+The resulting stability polynomial $P_{p;S}$ is then used to construct an optimized Runge-Kutta method.
+Trixi.jl implements the [Paired-Explicit Runge-Kutta (P-ERK)](https://doi.org/10.1016/j.jcp.2019.05.014) method, a low-storage, multirate-ready method with optimized domain of absolute stability.
+
+### Paired-Explicit Runge-Kutta (P-ERK) Schemes
+
+Paired-Explicit Runge-Kutta (P-ERK) or `PairedExplicitRK` schemes are an advanced class of numerical methods designed to efficiently solve ODEs.
+In the [original publication](https://doi.org/10.1016/j.jcp.2019.05.014), second-order schemes were introduced, which have been extended to [third](https://doi.org/10.1016/j.jcp.2022.111470)- and [fourth](https://doi.org/10.48550/arXiv.2408.05470)-order in subsequent works.
+
+By construction, P-ERK schemes are suited for integrating multirate systems, i.e., systems with varying characteristics speeds throughout the domain.
+Nevertheless, due to their optimized stability properties and low-storage nature, the P-ERK schemes are also highly efficient when applied as standalone methods.
+
+#### Tutorial: Using `PairedExplicitRK2`
+
+In this tutorial, we will demonstrate how you can use the second-order P-ERK time integrator.
+
+1. First, ensure you have the necessary packages installed. For the optimization of the stability polynomial $P_{2; S}$, you need the `Convex.jl` and `ECOS.jl` packages.
+You can install them using Julia's package manager:
+
+```julia
+using Pkg
+Pkg.add("Trixi")
+Pkg.add("Convex")
+Pkg.add("ECOS")
+```
+
+2. Now you can load the necessary packages:
+
+```julia
+using Convex, ECOS
+using Trixi
+```
```

Comment:
Can you please turn the code snippets into running code? Otherwise, it will get out of date and will become useless over time.

Reply:
I guess you could use `@example` (see [documentation](https://documenter.juliadocs.org/stable/man/syntax/#reference-at-example)).

Reply:
I have modified some parts to use `@setup`, `@repl`, and `@example` instead. Now when I build the documentation from (at least on my local PC) it added `Trixi`, `Convex`, and `ECOS` to `Project.toml`. I am not sure how to stop this from happening. Is this OK?

Reply:
So you say `Trixi` is added to the `Trixi.jl` `Project.toml` ? That sounds indeed not desirable.

Reply:
The main configuration file of the package is intact but `Trixi.jl` is added to `Project.toml` in `docs`.

Reply:
Adding Convex and ECOS to `docs/Project.toml` is fine. Please also provide appropriate compat bounds (e.g. take the ones from the main `Project.toml`). But you should not explicitly `Pkg.add` them (and also not Trixi.jl) in the docs pages. If they are in `docs/Project.toml` that is enough. Trixi.jl should point to the development version in `..` from `docs`, i.e. it will be added once you do `julia --project=docs -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()''` locally (or [in CI](https://github.com/trixi-framework/Trixi.jl/blob/3609f39b979de2e894bb78ecb9c67e8fe75ddf9f/.github/workflows/Documenter.yml#L46)). If you have Trixi.jl in `docs/Project.toml` locally you can just not push that.

Reply:
I see! I reckon the intention of this code block isn’t mainly to add the packages `Convex`, `ECOS`, and `Trixi` so the rest of the codes in the docs pages run fine but also to guide the user that these installations are necessary to use the optimized scheme. But if you don’t think it’s a good idea to add the packages in the documentation pages, I can also simply try to change these instructions to only be some texts instead.

Reply:
I think it's not necessary to explicitly add the packages in the documentation pages. We also don't do that for the other tutorials (regarding, e.g., OrdinaryDiffEq.jl or Plots.jl). You can, however, emphasize that one needs to add these packages. I'll make a suggestion.

Reply:
Got it! Thanks for the suggestion!